docs/source/man.rst

```diff
@@ -245,6 +245,10 @@ To create a source, supply the type of source with the ``type`` option, one or m
 
     ``--hosts /home/user1/hosts_file``
 
+``--exclude-hosts ip_address``
+
+  Sets the host name, IP address, or IP address range to exclude when running a scan. Follows the same formatting options as ``--hosts`` shown above.
```

Comment:
I would put `Optional.` at the beginning of the description so that we follow the flow of the other source options that aren't required.